mapping_form.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+
+<html lang="en">
```

Comment:
Thanks for taking the time and care to set up the proper `doctype`, `lang="en"` and `meta charset`. Very good to see this.